### PR TITLE
2.0.7

### PR DESCRIPTION
**Pull request by thecozies**

### CODE REVIEW

**Review by gheskett — APPROVED** [review]
Pre-approving because I fully expect this to be fine, but I still want to test it against gcc10 later before merging.

File: VERSION.txt [hunk 1]
```diff
@@ -1 +1 @@
-v2.0.6
+v2.0.7
```
Comment (someone2639) [inline VERSION.txt:1]: if you set this to 2.0.8 then you won't have to change it in the next PR

**Review by someone2639 — APPROVED** [review]

File: VERSION.txt (hunk 1 above)
Comment (Reonu) [reply VERSION.txt:1]: true!!!

**Comment by Reonu** [comment]
can't wait for 2.0.8!

**Pull request merged**